blockchain/fullblocktests/generate.go

```diff
@@ -1089,6 +1092,95 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	})
 	rejected(blockchain.ErrFreshStakeMismatch)
 
+	// Attempt to add block with tickets voting on the wrong block
```

Comment:
Total nitpick, but please add periods to these so they're consistent with the rest of the comments.